tools/cockpit.spec

```diff
@@ -644,12 +647,12 @@ cluster. Installed on the Kubernetes master. This package is not yet complete.
 %endif
 
 %package packagekit
-Summary: Cockpit user interface for package updates
+Summary: Cockpit user interface for packages
 Requires: %{name}-bridge >= %{required_base}
 Requires: PackageKit
 
 %description packagekit
-The Cockpit component for installing package updates, via PackageKit.
+The Cockpit components for installing packages, via PackageKit.
```

Comment:
I realize that we can't squeeze that into the short description, but the long description could point out that it's package updates or installing Cockpit Apps (not just any package).

Reply:
Done.